home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1704,7 +1710,7 @@ components:
         is_important:
           type: boolean
           example: true
-    CreateUser:
+    CreateUserDto:
```

Comment:
Dont we agreed that Dto suffix is for our service dtos, not for generated ones?

Reply:
Sasha leaved a comment in User's Crud Issue that we should use dto in naming
But you are right maybe he meant it for DTO that we use in services